### ISSUE TITLE
zbf file Export

### ISSUE THREAD

**Issue by tomflexcompute** [flexcompute/tidy3d#1005]
I'm wondering if we can do zbf file export to coupler FDTD with ray tracing in Zmax. Coupling full-wave to ray tracing seems to be a feature many people want/need. It seems like a relatively straightforward feature to implement since it only involves parsing the FieldMonitor data into a specific format as the example discussed here:
https://community.zemax.com/got-a-question-7/how-to-manually-create-a-zbf-file-941

Would you think we can implement this feature in the near future?

**Comment by lucas-flexcompute** [comment]
I'm looking into it, @tomflexcompute. It looks like it's doable, but there are a few settings in the zbf file that I'm don't yet fully understand.

**Comment by momchil-flex** [comment]
Hi @lucas-flexcompute this this fall through the cracks? I see there's some dangling commit. A user just asked about it.

**Comment by lucas-flexcompute** [comment]
No, we never had a chance to test it. The user who originally requested it ended up not really needing it, so it remained untested and unmerged. Recently, @tomflexcompute asked me to provide this functionality to another user and I split it into a separate function. Did they have any feedback, @tomflexcompute?

[Here](https://gist.github.com/lucas-flexcompute/ac8f8abc376f51222aad36beacb223ac)'s the function if it is of any interest.

**Comment by tomflexcompute** [comment]
Haven't heard anything yet. I can check with him early next week.

**Comment by yaugenst-flex** [comment]
Closing — ZBF export was implemented in PR #2397 (merged May 2025). Follow-up issue #2321 was also completed.
